src/Symfony/Component/Lock/Tests/Store/DoctrineDbalPostgreSqlStoreTest.php

```diff
@@ -59,4 +60,33 @@ public function getInvalidDrivers()
         yield ['sqlite:///tmp/foo.db'];
         yield [DriverManager::getConnection(['url' => 'sqlite:///tmp/foo.db'])];
     }
+
+    /**
+     * @requires extension pdo_sqlite
```

Comment:
should be removed

Reply:
Removed in https://github.com/symfony/symfony/pull/44828/commits/f25bd54d146899b45b7bbd70475c317eba543092